runtime/src/lib.rs

```diff
@@ -541,9 +544,11 @@ impl pallet_bounties::Config for Runtime {
     type BountyDepositBase = BountyDepositBase;
     type BountyDepositPayoutDelay = BountyDepositPayoutDelay;
     type BountyUpdatePeriod = BountyUpdatePeriod;
-    type BountyCuratorDeposit = BountyCuratorDeposit;
     type BountyValueMinimum = BountyValueMinimum;
     type DataDepositPerByte = DataDepositPerByte;
+    type CuratorDepositMin = BountyCuratorDepositMin; // TODO: put correct value here
+    type CuratorDepositMax = BountyCuratorDepositMax; // TODO: put correct value here
+    type CuratorDepositMultiplier = BountyCuratorDeposit; // TODO: put correct value here
```

Comment:
Please confirm these values from team and remove TODO comments after confirmation.

Reply:
Researching...